core/services/fluxmonitor/flux_monitor_test.go

```diff
@@ -1297,3 +1298,51 @@ func TestFluxMonitor_PollingDeviationChecker_HandlesNilLogs(t *testing.T) {
 		p.HandleLog(logBroadcast, nil)
 	})
 }
+
+func TestFluxMonitor_ConsumeLogBroadcast_Happy(t *testing.T) {
+	logBroadcast := new(mocks.LogBroadcast)
+	logBroadcast.On("WasAlreadyConsumed").Return(false, nil).Once()
+	logBroadcast.On("MarkConsumed").Return(nil).Once()
+
+	called := false
+	callback := func() {
+		if called == false {
+			called = true
+		} else {
+			panic("callback called more than once!")
```

Comment:
`t.Fatal("callback called more than once!")` is better than `panic` when it comes to tests.